internal/collector/processor/queued/queued_processor.go

```diff
@@ -149,6 +150,26 @@ func (sp *queuedSpanProcessor) processItemFromQueue(item *queueItem) {
 
 	// There was an error
 	statsTags := processor.StatsTagsForBatch(sp.name, processor.ServiceNameForNode(item.td.Node), item.td.SourceFormat)
+
+	// Immediately drop data on permanent errors. In this context permanent
+	// errors indicate some kind of bad data.
+	if errorkind.IsPermanent(err) {
+		numSpans := len(item.td.Spans)
+		sp.logger.Warn(
```

Comment:
A damper is always a good addition to logging (for observability in general). I didn't do it here because this specific change reduces the current amount of messages being logged. That said also worth adding a issue to track that feature: https://github.com/open-telemetry/opentelemetry-service/issues/94